python3Packages.botocore: 1.38.9 -> 1.38.32

dotlambda opened the pull request:
Diff: https://github.com/boto/botocore/compare/refs/tags/1.38.9...refs/tags/1.38.32

Changelog: https://github.com/boto/botocore/blob/1.38.32/CHANGELOG.rst
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [Nixpkgs 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/doc/release-notes/rl-2511.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/doc/manual/release-notes/rl-2505.section.md) Nixpkgs Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
- [NixOS 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2511.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) NixOS Release notes)
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

dotlambda committed:
python3Packages.botocore: 1.38.9 -> 1.38.32

Diff: https://github.com/boto/botocore/compare/refs/tags/1.38.9...refs/tags/1.38.32

Changelog: https://github.com/boto/botocore/blob/1.38.32/CHANGELOG.rst

dotlambda committed:
awscli2: 2.27.8 -> 2.27.31

Diff: https://github.com/aws/aws-cli/compare/refs/tags/2.27.8...refs/tags/2.27.31

Changelog: https://github.com/aws/aws-cli/blob/2.27.31/CHANGELOG.rst

mweinelt commented on pkgs/by-name/aw/awscli/package.nix:
```diff
@@ -14,14 +14,14 @@ let
     # N.B: if you change this, change botocore and boto3 to a matching version too
     # check e.g. https://github.com/aws/aws-cli/blob/1.33.21/setup.py
-    version = "1.40.8";
+    version = "2.27.31";
```
that makes awscli == awscli2

dotlambda replied:
Oops.

dotlambda committed:
awscli: 1.40.8 -> 1.40.31

Diff: https://github.com/aws/aws-cli/compare/refs/tags/1.40.8...refs/tags/1.40.31

Changelog: https://github.com/aws/aws-cli/blob/1.40.31/CHANGELOG.rst

dotlambda committed:
python3Packages.prompt-toolkit: fix importlib.metadata.PackageNotFoundError

dotlambda committed:
python3Packages.s3transfer: 0.12.0 -> 0.13.0

Diff: https://github.com/boto/s3transfer/compare/refs/tags/0.12.0...refs/tags/0.13.0

Changelog: https://github.com/boto/s3transfer/blob/0.13.0/CHANGELOG.rst

dotlambda commented:
I had to patch prompt-toolkit due to https://github.com/prompt-toolkit/python-prompt-toolkit/issues/1988. At least that patch should be merged into staging-next.

dotlambda commented:
@ofborg build python3.pkgs.moto
